### PR TITLE
Telegraf 1.19 new plugins

### LINKED ISSUE #2705: Telegraf 1.19 new plugins

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2705

### CODE REVIEW

File: data/telegraf_plugins.yml [hunk 1]
```diff
@@ -715,6 +745,13 @@ input:
     introduced: 1.17.0
     tags: [linux]
 
+  - name: Intel Data Plane Development Kit (DPDK)
+    id: dpdk
+    description: |
+      The DPDK plugin collects metrics exposed by applications built with Data Plane Development Kit, an extensive set of open source libraries designed for accelerating packet processing workloads.
+    introduced: 1.19
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:753]: networking

File: data/telegraf_plugins.yml [hunk 2]
```diff
@@ -840,6 +877,13 @@ input:
     introduced: 1.8.0
     tags: [linux, macos, windows, applications]
 
+  - name: KNX
+    id: knx_listener
+    description: |
+      The KNX input plugin that listens for messages on the KNX (Konnex) home-automation bus.
+    introduced: 1.19
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:885]: IoT

File: data/telegraf_plugins.yml [hunk 3]
```diff
@@ -1240,6 +1293,13 @@ input:
     introduced: 1.5.0
     tags: [linux, macos, windows, applications]
 
+  - name: OpenTelemetry
+    id: opentelemetry
+    description: |
+      Receives traces, metrics and logs from OpenTelemetry clients and agents via gRPC.
+    introduced: 1.19
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:1301]: logging, messaging

File: data/telegraf_plugins.yml [hunk 4]
```diff
@@ -1488,6 +1549,13 @@ input:
     introduced: 1.14.0
     tags: [linux, macos, windows, networking]
 
+  - name: Siemens S7 Comm
+    id: s7comm
+    description: |
+      This plugin gathers information from Siemens PLC (Programmatic Logic Controller).
+    introduced: 1.19.0
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:1557]: external, iot

File: data/telegraf_plugins.yml [hunk 5]
```diff
@@ -2191,6 +2275,13 @@ output:
     introduced: 1.9.0
     tags: [linux, macos, windows, cloud]
 
+  - name: SQL
+    id: sql
+    description: |
+      The SQL output plugin saves Telegraf metric data to an SQL database.
+    introduced: 1.19
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:2283]: data stores

File: data/telegraf_plugins.yml (hunk 1 above)
Comment (pierwill) [inline data/telegraf_plugins.yml:753]:
```suggestion
    tags: [networking]
```

File: data/telegraf_plugins.yml [hunk 6]
```diff
@@ -2224,6 +2315,13 @@ output:
     introduced: 1.5.0
     tags: [linux, macos, windows, applications, cloud]
 
+  - name: Websocket
+    id: websocket
+    description: |
+      The Websocket output plugin can write to a WebSocket endpoint.
+    introduced: 1.19
+    tags: []
```
Comment (sjwang90) [inline data/telegraf_plugins.yml:2323]: web, servers

**Pull request merged**